.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "doubtfire-web"]
 	path = doubtfire-web
-	url = https://github.com/doubtfire-lms/doubtfire-web.git
+	url = git@github.com:thoth-tech/doubtfire-web.git
 [submodule "doubtfire-api"]
 	path = doubtfire-api
-	url = https://github.com/doubtfire-lms/doubtfire-api.git
+	url = git@github.com:thoth-tech/doubtfire-api.git
 [submodule "doubtfire-overseer"]
 	path = doubtfire-overseer
 	url = https://github.com/doubtfire-lms/doubtfire-overseer.git
```

Comment:
**suggestion:** I have forked this repo to Thoth-tech. For consistency, we can update this to point to the Thoth-tech [fork](https://github.com/thoth-tech/doubtfire-overseer) as well.